--- src/scim_table_imengine_setup.cpp~	2005-10-26 09:53:53.000000000 +0200
+++ src/scim_table_imengine_setup.cpp	2009-01-21 20:06:33.000000000 +0100
@@ -2326,6 +2326,7 @@
                                           GTK_DIALOG_MODAL,
                                           GTK_MESSAGE_ERROR,
                                           GTK_BUTTONS_CLOSE,
+                                          "%s",
                                           err.c_str ());
         gtk_dialog_run (GTK_DIALOG (msg));
         gtk_widget_destroy (msg);
